DevoxxClientMobile/src/main/java/com/devoxx/views/helper/FilterSessionsPresenter.java

```diff
@@ -226,6 +226,18 @@ public void initialize() {
         updateSearchPredicate();
     }
 
+    public void hidePastSession() {
+        periodAll.setSelected(false);
+        for (Node child : periodRadioBtContainer.getChildren()) {
```

Comment:
Use `radioButtonGroup` instead of periodRadioBtContainer.

```
for (Toggle toggle : radioButtonGroup.getToggles()) {
    if (toggle.getUserData() == TimePeriod.MORE_THAN_ONE_HOUR_AGO) {
        toggle.setSelected(true);
        return;
    }
}
```